packages/clarity/src/core/client.ts

```diff
@@ -71,8 +83,8 @@ export class Client {
       this.name,
       `(${query.method.name} ${query.method.args.join(" ")})`,
       true,
-      query.atChaintip,
+      query.atChaintip
     );
```

Comment:
It looks like this is an extra closing parenthesis on line 87 of the new code, based on commenting out the `let res = await` line?